samples/app/MeshAssimp.cpp

```diff
@@ -114,74 +113,40 @@ std::string shaderFromConfig(MaterialConfig config) {
         void material(inout MaterialInputs material) {
     )SHADER";
 
-    if (!config.unlit) {
-        if (config.normalUV == 0) {
-            shader += R"SHADER(
-                material.normal = texture(materialParams_normalMap, getUV0()).xyz * 2.0 - 1.0;
-                material.normal.y = -material.normal.y;
-            )SHADER";
-        } else {
-            shader += R"SHADER(
-                material.normal = texture(materialParams_normalMap, getUV1()).xyz * 2.0 - 1.0;
-                material.normal.y = -material.normal.y;
-            )SHADER";
-        }
-    }
+    shader += "float2 normalUV = " + std::string(config.normalUV == 0 ? "getUV0()" : "getUV1()") + ";\n";
```

Comment:
You could write `"float2 normalUV = getUV" + std::to_string(config.normalUV) + "();\n"` instead. It's simpler :)